code/datums/martial/cqc.dm

```diff
@@ -80,9 +80,21 @@
 		return TRUE
 
 /datum/martial_art/cqc/proc/Kick(mob/living/attacker, mob/living/defender)
-	if(!can_use(attacker))
+	if(!can_use(attacker) || defender.stat == DEAD)
 		return FALSE
-	if(!defender.stat || !defender.IsParalyzed())
+
+	var/helmet_protection = defender.run_armor_check(BODY_ZONE_HEAD, MELEE)
+	var/knockout_prob = rand(-15,15) + defender.getStaminaLoss() - helmet_protection
```

Comment:
This probability is almost always zero in any instance that it matters. Let me explain.

CQC does predominantly Brute (and very rarely burn if playing an ethereal). For CQC to do Stamina damage you either need to be;

A) Doing the Pressure combo, which is just awful. Like, literally better off doing the Slam combo for that many inputs if anything.
B) Doing the Consecutive CQC combo, which means the other person has their brain switched off as you hit them two times with a shove and then a harm and they didn't manage to somehow stop you from doing that. And that also has no attached disable, so you'll have to do that again.
C) You are using the Restrain combo, in which case they are already dead because you can just knock them out using that combo. You are doing the _right_ thing.

Since the sources of stamina damage are distinctly poorly designed, attaching the random probability of a knockout that already requires the target to be prone is absurd. The two moves that do predominantly Stamina damage do not make the target prone. The one that does already knocks them out without an RNG chance attached.

This then leads into the issue of 'okay, but what if I gamble on it anyway? I got the legsweep, so let's just go for the head kick.'

If the target has even the most basic security helmet, that's a -35 percent chance. You have also added a negative range on your random probability without a minimum value that is more than the possible negative (something around 20% let's say), which means that more than half the time, it just is 0%. If you've done at least one of the stamina moves on your target, you'll likely have an okay chance, but if you could perform any of those moves, why have you not used a move that just ends the fight then and there with a hard knockout? If they have a helmet at any point, they are almost assuredly safe from the knockout.

Reply:
I welcome you to balancing CQC; it's very badly designed. You're gonna have to rethink the _whole thing_, my dude.